lib/reversetunnel/localsite.go

```diff
@@ -634,6 +634,22 @@ func (s *localSite) sshTunnelStats() error {
 			return false
 		}
 
+		ids := server.GetProxyIDs()
+		containsID := false
+		for _, id := range ids {
+			if id == s.srv.ID {
+				containsID = true
+				break
+			}
+		}
+
+		// A node is expected to be connected to the current proxy if the proxy
+		// id is present. A node is expected to be connected to all proxies if
+		// no proxy ids are present.
+		if !(containsID || len(ids) == 0) {
+			return false
+		}
```

Comment:
I wonder if the following reads better:
```suggestion
		// Thus, skip server if it has proxy ids and the proxy id is not there.
		if len(ids) > 0 && !containsID {
			return false
		}
```

Reply:
```suggestion
		if len(ids) != 0 && !slices.Contains(ids, s.srv.ID) {
			return false
		}
```

with `slices` from `golang.org/x/exp/slices`.